README.md

```diff
@@ -52,6 +52,11 @@ start_appinventor
 
 Press Ctrl+C to quit the server. Enter exit at the prompt to leave the virtual machine. To reclaim resources when you are not actively developing, you can run `vagrant halt` to stop the virtual machine. To completely remove the virtual machine, run `vagrant destroy`. If you destroy the VM, you will need to start these instructions from the top.
 
+Note 1: If it seems like none of the dependencies are installed in the VM, run ```vagrant provision```.
+
+Note 2: For mac users, if you are using virtualbox and any get error while initializing the VM its probably a due to system preferences, consider reading [this](https://medium.com/@Aenon/mac-virtualbox-kernel-driver-error-df39e7e10cd8) article. 
```

Comment:
A couple changes:

* mac => macOS
* virtualbox => VirtualBox
* any get error => get any error
* its probably a due to system preferences => it may be due to security restrictions in System Preferences

And I would say switch the order of note 1 and 2, since the conditions of note 1 are a symptom of the failure specified in note 2.

Reply:
@ewpatton Done.